1_fetch/src/get_nwis_data.R

```diff
@@ -1,48 +1,33 @@
+# Download site data from USGS site file site
+# Save output as CSV
+download_nwis_site_data <- function(site_num, fileout, parameterCd = '00010', startDate="2014-05-01", endDate="2015-05-01"){
```

Comment:
It's nice to have some consistency between functions in a single pipeline who have similar arguments. You've done part of that here - naming all filepath arguments `fileout` in each of your functions - but one other thing you can do is be consistent with the order of those shared arguments. One common thing that we do with file args is always put them at the beginning of our arg list (both in and out file args). You've already done that for every function except this one, so I am suggesting you swap the order of `site_num` and `fileout` to match the other functions! You would need to change the targets that call this function by editing `_targets.R`, too.